src/applications/disability-benefits/all-claims/tests/migrations/migrations.unit.spec.js

```diff
@@ -237,4 +238,75 @@ describe('526 v2 migrations', () => {
       expect(migratedData).to.deep.equal(getData(Object.values(list)));
     });
   });
+
+  describe('08-paper-sync', () => {
+    it('should not change returnUrl if user left off on the veteran info page', () => {
+      const savedData = {
+        formData: {},
+        metadata: {
+          returnUrl: '/veteran-information',
+        },
+      };
+      const migratedData = reorderHousingIllnessRemoveFdc(savedData);
+      expect(migratedData.metadata.returnUrl).to.equal('/veteran-information');
```

Comment:
I'd add `version` to the `savedData`, then use `.to.deep.equal()` here to ensure the other metadata keys are preserved.